src/pages/index.tsx

```diff
@@ -82,18 +82,30 @@ const SocialCards: SocialCardProps[] = [
     children: (
       <img
         alt="reddit"
-        className="m-auto h-20"
+        className="m-auto h-14"
         src="/img/homepage/Reddit-Logo-White.svg"
       />
     ),
   },
+  {
+    color: "bg-[#563ACC]",
+    link: "https://mastodon.social/@meshtastic",
+    isMeLink: true,
+    children: (
+      <img
+        alt="mastodon"
+        className="m-auto h-10"
+        src="/img/homepage/logo-white.svg"
```

Comment:
Would be nice to have this file be named mastodon-logo or something more specific 🍺